addons/point_of_sale/tools/posbox/overwrite_before_init/etc/systemd/system/odoo.service

```diff
@@ -6,6 +6,7 @@ Wants=network-online.target
 [Service]
 User=pi
 Group=pi
+Environment="LIBCAMERA_LOG_LEVELS=3"
```

Comment:
we could also use `EnvironmentFile=` and set a path to a file in `iot_box_image/configuration` in order to be able to update it with a checkout and a restart.